### ISSUE TITLE
Comments dictionary changed format

### ISSUE THREAD

**Issue by alvin-leong** [minimaxir/facebook-page-post-scraper#98]
Facebook seems to have removed the "from" property from Comments. This causes a problem for the comment_author field in the comments scraper. Is there a possible workaround?

**Comment by iamrajeshchauhan** [comment]
That's true me too getting same error

**Comment by philippnkling** [comment]
For me, the same error occurs. Will someone adapt the existing script in the next weeks?

**Comment by matthiasfru** [comment]
Just FYI, as of 06/02, getting User information now requires a page access token (you need to be admin of that page afaik). 
It appears that this change is **permanent for all versions of the API**.
see https://developers.facebook.com/ads/blog/post/2017/11/07/marketing-api-v211/
So I guess the "workaround" is to replace
```python
comment_author = unicode_decode(comment['from']['name'])
``` 
with 
```python
comment_author = None
```

**Comment by leduyloc** [comment]
Yes, maybe because many competitive pages try to scan user's information of other pages so facebook locks this feature.

**Comment by wachmann** [comment]
Thanks, @matthiasfru. That really sucks though :/

**Comment by iamrajeshchauhan** [comment]
Hi @matthiasfru , Thanks for the twisting the code. 

However, is there any other source to get the comment authour name ???
